origins_workflow/origins_workflow.install

```diff
@@ -13,6 +13,11 @@ use Drupal\user\Entity\User;
  * Perform actions to set up the site for this module.
  */
 function origins_workflow_install() {
+  // Assign a weight 1 higher than pathauto to ensure that alias
+  // handling in origins_workflow_entity_update is not negated
+  // by post save processing in pathauto.
+  module_set_weight('origins_workflow', 2);
```

Comment:
Does this duplicate/potentially conflict with what is (I think?) now set in core.extension.yml?

Reply:
Well, my feeling was that we need both!
When you initially install the module the weight is set to 2, so this would be good for when we are setting up new Unity sites, for example.
But the weight also needs to be set to 2 in core.extension.yml, otherwise we will set the weight to zero every time that we run config import, which we don't want.
So I think that we need to have it set here for installing on new sites, but then we need to set it in core.extension as well on each site that uses it.